schema/properties_directory/structural/patterns/special_bond.json

```diff
@@ -0,0 +1,28 @@
+{
+    "schemaId": "properties-directory-structural-pattern-special-bond",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "special bond pattern schema",
```

Comment:
We should provide a bit more explanation (description field?) regarding what we treat as a "special_bond", perhaps in the schema otherwise it is a bit unclear. From the example above it is hard to guess the meaning either.